### DISCUSSION TITLE
XeLaTeX confusion

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#145] [Q&A] [answered]
I normally use just the commands like this
`\newfontfamily{\mufi}{Junicode Two Beta}` and it works.
For the TUGboat paper I had to use commands like this
`\newfontfamily{\Junicode}{JunicodeTwoBeta-Regular.ttf}[Path=/home/jsbien/work/Junicode-font/fonts/ttf/]`
where `Junicode-font` is just a clone of the repository; all the files of the `fonts` subdirectory are linked to `./local/shared/fonts`.
I got the source of the proof with just
`\newfontfamily{\Junicode}{JunicodeTwoBeta-Regular.ttf}`
and it doesn't work for me.
I'm afraid I'm unable to understand fontspec documentation, in my opinion it is quite confusing, Google search doesn't help. I'm a subscriber of the XeTeX mailing list and I can ask a question there, but first I would like to get some idea what is the relation between names like `Junicode Two Beta` and file names like `JunicodeTwoBeta-Regular.ttf`.
I will appreciate your comments.

**Comment by psb1558** [comment, marked as answer]
I don't understand everything in the fontspec documentation either: I usually figure out one thing that works for me and just stick with that. My impression is that when a filename and no Path is given, fontspec will search in TeX's default search paths. I don't know much about TeX searching, but I know that the font file will always be found when it's in the same directory as your XeLaTeX source. Presumably the person who worked with your file put the Junicode files in the same directory and altered the source to reflect that.

I see no reason not to put it back the way you had it before.

**Reply by jsbien** [reply]
>  the font file will always be found when it's in the same directory as your XeLaTeX source. 

Thank for reminding me.
